### ISSUE TITLE
Version 1.12.1 macOS Ventura fails to start on fresh install

### ISSUE THREAD

**Issue by bxbrenden** [rancher-sandbox/rancher-desktop#6329]
### Actual Behavior

I installed Rancher Desktop `1.12.1` for Apple Silicon from the download link on `https://rancherdesktop.io/`, and when I start Rancher Desktop, I am immediately greeted with a pop-up error message in the UI that says there was a Kubernetes issue and that the Lima VM cannot start because it was sent a `SIGKILL` signal from my OS. The Rancher Desktop UI stays running, and I can interact with it; however, the Docker daemon never starts, so I can't run any containers despite the status in the UI saying "Online".

I also tried running `rdctl shell` while Rancher Desktop was running, and I was given the same error from the command line:
```
$ rdctl shell
ERRO[0000] Failed to run "/Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl.ventura ls 0 --format {{.Status}}": signal: killed
```

To test whether this was a version issue, I installed Rancher Desktop `1.11.1` from the releases page on GitHub, and it is working perfectly well.

### Steps to Reproduce

1. Download the `aarch64` version of Rancher Desktop `1.12.1` from https://rancherdesktop.io/ as a `.dmg` file.
2. Double-click the `.dmg` file from Finder.
3. When prompted to do so, drag and drop the Rancher Desktop executable into the `Applications` folder to copy it to your applications.
4. Once the copying is completed, double-click Rancher Desktop from the `Applications` folder to start it
5. Choose the default settings and ensure you don't enable Kubernetes (though it doesn't seem to matter whether or not you enable K8s for this).
6. You will be greeted with a Kubernetes "received SIGKILL" error once you are past the setup screen from step 5.

### Result

The Docker daemon never starts, and 

```
$ rdctl shell
ERRO[0000] Failed to run "/Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl.ventura ls 0 --format {{.Status}}": signal: killed
```

### Expected Behavior

Rancher Desktop should start without issue

### Additional Information

I also tried using the "Factory Reset" and "Reset Kubernetes" options under the `Troubleshooting` section of the UI, but they did not help.

### Rancher Desktop Version

1.12.1

### Rancher Desktop K8s Version

N/A

### Which container engine are you using?

moby (docker cli)

### What operating system are you using?

macOS

### Operating System / Build Version

macOS Ventura 13.0

### What CPU architecture are you using?

arm64 (Apple Silicon)

### Linux only: what package format did you use to install Rancher Desktop?

None

**Comment by jandubois** [comment]
I cannot reproduce this (on M1 with macOS Sonoma).

Can you run the command manually, to see if you get any additional output?

```
"/Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl.ventura" ls 0 --format "{{.Status}}"
```

**Comment by LEEJIYUL** [comment]
i have same problem! did you resolve this issue?

**Comment by abaerptc** [comment]
Same issue here with macOS x86_64. Multiple reinstalls and factory resets did not work. Downgraded to 1.11.1 and everything worked again.

**Comment by jandubois** [comment]
There is not much we can do unless the people affected provide the full set of log files, plus information about OS version, CPU arch, etc.

So please, if this is reproducible by you, collect all the logs in the log directory and attach them to a bug report or make them otherwise available.

**Comment by Zacharius** [comment]
[ha.stdout.log](https://github.com/rancher-sandbox/rancher-desktop/files/14088037/ha.stdout.log)

I am experiencing a very similar issue. I believe I've attached the main error file

**Comment by jandubois** [comment]
> I am experiencing a very similar issue. I believe I've attached the main error file

There is no relevant information in this file. It seems like the VM is not running after 2 minutes, which is long, but still possible depending on the machine and load. Please wait for more than 2 minutes and attach a ZIP file with the full set of logs!

**Comment by Zacharius** [comment]
So I just updated to the newest version of macOS, Sonoma 14.3, and that appears to have resolved the issue.

**Comment by lng-in2** [comment]
Same issue here and solved it in the same way as @Zacharius 🤷

**Comment by lbeckman314** [comment]
Hmm... seem to be encountering the same issue, will try downgrading to 1.11.1 as @abaerptc recommended.

![rancher-desktop-error](https://github.com/rancher-sandbox/rancher-desktop/assets/15309567/bc760f9b-5702-42a6-8d14-7eb12d4648cb)

## Log Files

@jandubois I've just attached a zip file with all logs from `~/Library/Logs/rancher-desktop/`, let me know if any additional logs would be helpful!

> Rancher Desktop Logs: [rancher-desktop-logs.zip](https://github.com/rancher-sandbox/rancher-desktop/files/14349977/rancher-desktop-logs.zip)

## OS Info
- Chip: Apple M1 Max
- macOS: Venture 13.2 (22D49)

**Comment by fseesink** [comment]
Experiencing same issue on two (2) new Macs:
- M3 iMac w/ 24GB RAM
- M3MAX 16" MacBook Pro w/ 64GB RAM
- both running macOS "Sonoma" v14.3.1
- Rancher Desktop 1.12.3

(This appears to be the same issue, hence reporting here.  But if you prefer this opened as a new issue, please let me know.)

This makes me think whatever is going on is not architecture specific, nor necessarily solely with 1.12.1.

Mind you these are new Macs.  So clean installs of both macOS 14.x and Rancher Desktop 1.12.3.

After getting error, rebooted system just to see if possibly something needed to be put in place (e.g., the files in `~/.rd/bin/`).  But both system exhibit same behavior.

On firing up Rancher Desktop, they take easily an excruciating 7+ minutes to try and start, eventually showing "Starting virtual machine" in the lower right with the progress bar, and at some point after that kicking up the error as shown in this screenshot take on the M3 iMac (which took nearly 9 minutes):
<img width="1348" alt="Screenshot 2024-02-22 at 11 59 39 AM" src="https://github.com/rancher-sandbox/rancher-desktop/assets/48991/7d59e01c-e184-4f08-9c47-7063756aeec2">

Also attaching the log files (in `.tar.gz` format) from both systems.

I know this is not normal behavior, as I have used Rancher Desktop on a 2019 27" iMac with an Intel Core i7 w/ 32GB RAM for well over a year now.  And usually startup time is ~1.5 minutes give or take.  This shift to the new M3 Macs I knew would bring quite some changes going from x64 to ARM64.  So I did a clean install, not migrating any files/etc.  Hence why I say a clean install, as it's both a new OS (fresh, not upgraded from "Ventura" etc.) and a new install of Rancher Desktop's latest version.

If there is anything else I can provide, please don't hesitate to ask.
[rancherlogs-imac.tar.gz](https://github.com/rancher-sandbox/rancher-desktop/files/14377069/rancherlogs-imac.tar.gz)
[rancherlogs-macbookpro.tar.gz](https://github.com/rancher-sandbox/rancher-desktop/files/14377070/rancherlogs-macbookpro.tar.gz)

As I have yet to begin working with this Rancher Desktop setup, if you need any testing done, please let me know.  I would very much like to get to something usable.

**Comment by fseesink** [comment]
It should be noted that after clicking on `[Close]` above and then quitting Rancher Desktop, it took ~3 minutes for each one (on both M3 iMac and M3MAX MBP) to finally shut down.

**Comment by fseesink** [comment]
Not sure whether this helps or not, but after installing Rancher Desktop and trying to start it, I did modify the default configuration as follows to match what I had on my 2019 iMac:

In Preferences | Application, under General tab:
[x] Allow to acquire administrative credentials (sudo access)
[ ] Check for updates automatically
[ ] Allow collection of anonymous statistics...

In Preferences | Application, under Environment tab:
   Configure PATH (*) Manual
 
In Preferences | Virtual Machine
- 8 GB (iMac) / 16 GB (MBP)
- 3 CPUs (iMac) / 4 CPUs (MBP)

In Preferences | Container Engine
 (*) `containerd`

I left the rest alone, such as the Kubernetes version (`1.28.6 (stable)`), etc.

**Comment by aeimer** [comment]
Hi guys, I recently updated to Sonoma 14.6.1 and I get this error with 1.14.1 and 1.15.1:

```
'--tty=false',
    '0'
  ],
  stdout: '',
  stderr: 'time="2024-08-28T17:40:55+01:00" level=info msg="Using the existing instance \\"0\\""\n' +
    'time="2024-08-28T17:40:55+01:00" level=fatal msg="errors inspecting instance: [failed to get Info from \\"/Users/a/Library/Application Support/rancher-desktop/lima/0/ha.sock\\": Get \\"http://lima-hostagent/v1/info\\": dial unix /Users/a/Library/Application Support/rancher-desktop/lima/0/ha.sock: connect: connection refused]"\n',
  code: 1,
  [Symbol(child-process.command)]: '/Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl.ventura start --tty=false 0'
}
```

Is there any fix to that?

**Comment by kbanach** [comment]
@aeimer I had the issue for a while, couldn't find anything in google, but switching Virtual Machine type to VZ solved the issue for me for some reason 🤷‍♂️

<img width="450" alt="image" src="https://github.com/user-attachments/assets/f50a36cd-0ce4-4745-bbfe-95ff0aee6665">

**Comment by jandubois** [comment]
This issue seems expired; the failure to start on M3 chips was fixed in the 1.13.0 release.

For the comments about other issues that have been attached here, please file separate issues with full logs.
